dev/com.ibm.ws.kernel.metatype.helper/src/com/ibm/wsspi/config/internal/FilesetImpl.java

```diff
@@ -541,4 +573,29 @@ private void notifyListeners() {
         }
     }
 
+    @Override
+    @FFDCIgnore(Exception.class)
+    public String toString() {
+        try {
+
+            StringBuilder sb = new StringBuilder();
+
+            sb.append("FileSet for directory: " + basedir);
+            sb.append(System.lineSeparator());
+
+            if (filter.hasFilters()) {
+                sb.append(filter.getHumanReadableSummary()); //provides its own line break
+            }
+
+            String files = fileset.stream().map(File::getPath).collect(Collectors.joining(", "));
```

Comment:
quibble: use the buffer inside the string builder instead of concatenating strings in that collector.
```
fileset.stream().limit(1).map(File::getPath).forEach(sb::append)
fileset.stream().skip(1).map(File::getPath).forEach(p -> sb.append(", ").append(p))
```